tonto/routers_tonto.py

```diff
@@ -0,0 +1,37 @@
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+import cowboys
+from tonto import crud, schemas
+from settings import bd_coneccion
+
+router = APIRouter()
+
+
+@router.get ("/tonto", response_model = cowboys.schemas.CrearCowboyRespuesta)
+def seleccionar_tonto (db: Session = Depends (bd_coneccion.get_db)):
+    cowboy  = crud.tonto_random (db)
+
+    if cowboy is None:
+        raise HTTPException (status_code = 404, detail = "No existe ese cowboy")
+
+    return cowboy
+
+
+@router.post ("/tonto", response_model = schemas.TontoRespuesta)
```

Comment:
```suggestion
@router.post ("/tontos", response_model = schemas.TontoRespuesta)
```

Se tiene que llamar `/tontos` para que sea compatible con la api actual.